frontend/sentiment-dashboard/src/app/settings/settings.component.html

```diff
@@ -1,3 +1,94 @@
-<p>
-  settings works!
-</p>
+<h1>Settings and Administration</h1>
+
+<div class="row">
+  <h2 class="h4">App Management</h2>
+  <button type="button" class="btn btn-sm btn-primary ml-auto">Add App</button>
+</div>
+
+  <div class="form-group">
+
+  </div>
+<hr>
+
+
+<div class="row align-items-start font-weight-bold">
```

Comment:
Can we use a table here instead of a bunch of divs? Does Bootstrap provide classes we can use to style a table?